www/js/app.js

```diff
@@ -267,17 +267,17 @@ $('#searchArticles').on('click', function () {
     // This flag is set to true in the mousedown event below
     searchArticlesFocused = false;
 });
-$('#searchArticles').on('mousedown', function () {
+document.getElementById('searchArticles').addEventListener('mousedown', function () {
     // We set the flag so that the blur event of #prefix can know that the searchArticles button has been clicked
     searchArticlesFocused = true;
 });
-$('#formArticleSearch').on('submit', function () {
+document.getElementById('formArticleSearch').addEventListener('submit', function () {
     document.getElementById('searchArticles').click();
     return false;
```

Comment:
`return false` is jQuery syntax. Please use the vanilla JS equivalent, of omit if not needed.